### PR TITLE
added two blogs

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

Related to #519 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jashpatel1** [comment]
> These images aren't aligned properly 
> Screenshot from 2020-08-04 00-16-57
> 

They are intentionally kept to fit two images in a single row reducing the scrolling length.

**Comment by Jashpatel1** [comment]
> > These images aren't aligned properly
> > Screenshot from 2020-08-04 00-16-57
> 
> They are intentionally kept to fit two images in a single row reducing the scrolling length.

Will do it.

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import linkerd from './linkerd.svg';
+import logo from './logo.svg';
+import nsm from './nsm.svg';
+import octarine from './octarine.svg';
+
+<BlogWrapper>
+
+<span class="starting-letter">I</span>t’s no secret that service mesh tech is boiling hot. Microservice architectures brought on as many challenges as they have advantages. With operational complexity being one of the most acute pains. Service meshes do offer solutions to a number of these operational concerns. Including but not limited to: resilience, improved observability, security and advanced service discovery.
+<div class="row">
+  <div class="col m8">
```
Comment (Tanuj22) [inline src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:24]: `col m8` and `row` class are not being used now I think

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+<li>What service mesh tech allows us to do</li>
+<li>What a typical service mesh architecture looks like</li>
+<li>What Layer5 is about (<i>Lookout - it may surprise you!</i> 😯)</li>
+<li>What Meshery is. What Linkerd is.</li>
+<li>How easy it is to install Meshery on your PC (be it Linux, Mac or Windows)</li>
+<ul><li>All it takes is: 
+
+```sh
+$ curl -L https://git.io/meshery | bash -
```
Comment (Tanuj22) [inline src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:122]: The background is transparent for some reason.
Screenshot from 2020-08-04 22-53-55

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-app-mesh">
+                <img src={awsappmesh} alt='AWS App Mesh Service Mesh' class="adapter-logo" />Meshery adapter for App Mesh </a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-kuma">
+                <img src='https://pbs.twimg.com/profile_images/1171109603158843392/bV2M7l39_400x400.jpg' alt='Kuma Service Mesh' class="adapter-logo" />Meshery adapter for Kuma</a>
```
Comment (Tanuj22) [inline src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:94]: This image is not available.

Comment (Jashpatel1) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:94]: This image is not available in the current site as well.

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx (hunk 2 above)
Comment (Jashpatel1) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:122]: Strange. It works fine with me.

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx (hunk 3 above)
Comment (Tanuj22) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:94]: [Here]( https://layer5.io/assets/images/landscape/Kuma.svg) you go. We should also open up an issue for the existing site I think

Comment (Jashpatel1) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:94]: Cool, I will do it.

File: src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx (hunk 2 above)
Comment (Tanuj22) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:123]: This looks like a bigger problem. The background is not loading on refresh the page. Can you verify @Nikhil-Ladha

Comment (Nikhil-Ladha) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:123]: Indeed it is @Tanuj22 initially the background loads, but then after a refresh ir page change it doesn't comes back. To confirm, I saw the blog from Kush as well `Getting started with mesheryctl` it also has a similar problem.

Comment (Jashpatel1) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:123]: This seems to be an issue with netlify, as the local site works perfectly on reload.

Comment (Tanuj22) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:123]: As @Jashpatel1 said it is working locally. I think I have a solution for this in production. We can move ahead with this PR @Nikhil-Ladha . I will open a new PR for the solution

Comment (Nikhil-Ladha) [reply src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx:123]: Cool.

**Comment by leecalcote** [comment]
Sweet.

**Pull request merged**